src/patternfly/components/Page/page.scss

```diff
@@ -206,6 +212,29 @@
   @media screen and (min-width: $pf-global--breakpoint--lg) {
     grid-template-columns: auto 1fr auto;
   }
+
+  .pf-c-dropdown {
+    .pf-c-avatar {
+      --pf-c-page__header-tools--c-avatar--MarginLeft: 0;
+    }
+  }
+
+  .pf-c-dropdown__toggle {
```

Comment:
I wonder if we should just make the dropdown enhancement, then the page enhancements once we finish the overall masthead story? Some questions:

* If someone is using another dropdown in the header, do we want the bottom border styles applied?
* Would we want the underline styles to apply to the existing user dropdown without the avatar?
* Can we handle the bottom border better so that the sides aren't angled?
* Since we can include this in the breaking release, do we want to support using the avatar outside of the dropdown menu anymore? 

Here's a screenshot of another dropdown in the header with the underline styles, also showing the angled sides.

<img width="471" alt="Screen Shot 2020-03-10 at 5 50 09 PM" src="https://user-images.githubusercontent.com/35148959/76366462-0e53b600-62f8-11ea-8067-fbcab0550853.png">

Reply:
Agree. #2808

Reply:
These are all good questions @mcoker . I know that we talked a bit about this last week. I think we were leaning towards making all dropdowns in the masthead have the same appearance, but that would still be unique to the masthead. Do you recall specifics @mceledonia ?